src/views/AmalgamationSelection.vue

```diff
@@ -224,7 +224,11 @@ export default class AmalgamationSelection extends Vue {
       type === AmalgamationTypes.VERTICAL ? this.getBusinessEmail : ''
     const phone = type === AmalgamationTypes.HORIZONTAL ||
       type === AmalgamationTypes.VERTICAL ? this.getFullPhoneNumber : ''
-
+    const correctNameOption =
+      type === AmalgamationTypes.HORIZONTAL ||
+      type === AmalgamationTypes.VERTICAL
+        ? CorrectNameOptions.CORRECT_AML_ADOPT
+        : null
```

Comment:
Please test this (locally or using the preview URL). I tried a few times and it looks like the correct name option isn't been saved in the draft amalgamation.

![image](https://github.com/bcgov/business-filings-ui/assets/10002889/98402149-3302-4987-8cbc-7e77df9d0a93)

Reply:
Resolved by updating shared files enums